src/redux/constants/viewPaginationConstants.js

```diff
@@ -1,8 +1,4 @@
-const VIEW_FIRST_PAGE = 'VIEW_FIRST_PAGE';
-const VIEW_LAST_PAGE = 'VIEW_LAST_PAGE';
-const VIEW_PREVIOUS_PAGE = 'VIEW_PREVIOUS_PAGE';
-const VIEW_NEXT_PAGE = 'VIEW_NEXT_PAGE';
-const VIEW_PAGE_NUMBER = 'VIEW_PAGE_NUMBER';
+const VIEW_PAGE = 'VIEW_PAGE';
 const SET_PER_PAGE = 'SET_PER_PAGE';
 
-export { VIEW_FIRST_PAGE, VIEW_LAST_PAGE, VIEW_PREVIOUS_PAGE, VIEW_NEXT_PAGE, VIEW_PAGE_NUMBER, SET_PER_PAGE };
+export { VIEW_PAGE, SET_PER_PAGE };
```

Comment:
required:
the new linting updates require their to be a default here... this file should be updated to reflect that
```
...
const viewPaginationTypes = {
  SET_PER_PAGE,
  VIEW_PAGE
};

export { viewPaginationTypes as default, viewPaginationTypes, SET_PER_PAGE, VIEW_PAGE };
```

Reply:
Updated, thanks!